onnxscript/ir/passes/common/lift_constants_to_initializers.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Lift constants to initializers."""
+
+from __future__ import annotations
+
+__all__ = [
+    "LiftConstantsToInitializersPass",
+]
+
+import logging
+
+from onnxscript import ir
+
+logger = logging.getLogger(__name__)
+
+
+class LiftConstantsToInitializersPass(ir.passes.InPlacePass):
+    def call(self, model: ir.Model) -> ir.passes.PassResult:
+        """Convert constant nodes in main graph to initializers."""
+        count = 0
+        for node in model.graph:
+            if node.op_type != "Constant":
+                continue
+            if "value" not in node.attributes:
```

Comment:
This doesn't handle all the variations supported by the Constant op. I think you may be able to use [this utility function](https://github.com/microsoft/onnxscript/blob/2962a09e5b4dd5e0e5afc2e975f66c8bac2452dd/onnxscript/optimizer/_constant_folding.py#L65), but it is in the optimizer module, may need to move it to a common usable place.

Reply:
Done